### ISSUE TITLE
how define the loss function if I apply capsule network in regression task? 

### ISSUE THREAD

**Issue by fangxu622** [Moeinh77/Matrix-Capsule-Networks-PyTorch#1]
thanks for sharing your capsule implementation.

Any suggestions about loss function for the regression task?
